src/components/BreadcrumbNav/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const BreadcrumbNav = ({ list }) => {
```

Comment:
Can we memoize this component

Reply:
Let's pick that up if there is a performance issue? I prefer optimizations when we know they are needed.